static/include/snippet/show-secret.md

```diff
@@ -1,9 +1,9 @@
-{{< alert title="Location secret" color="info" >}}
-By default, the sample code does not include your robot location secret.
-We strongly recommend that you add your location secret as an environment variable and import this variable into your development environment as needed.
+{{< alert title="API key and API key id" color="info" >}}
+By default, the sample code does not include your robot API key and API key id.
+We strongly recommend that you add your API key and API key id as an environment variable and import this variable into your development environment as needed.
 
-To show your robot's location secret in the sample code, toggle **Include secret** on the **Code sample** tab.
-You can also see your location secret on the [locations page](/manage/fleet/locations/).
+To show your robot's API key and API key id in the sample code, toggle **Include secret** on the **Code sample** tab.
+You can also see your API key and API key id on your robot's **Security** tab.
 {{< /alert >}}
```

Comment:
Would suggest making these all lowercase or keeping other parts uppercase (api vs API) but not required. I think API looks slightly better but its lowercase in a lot of other places in the docs

Reply:
not on the API pages haha - so let's standardize to uppercase